niceplots/utils.py

```diff
@@ -624,6 +625,167 @@ def plotColoredLine(
         return
 
 
+def plotNestedPie(
+    data,
+    colors=None,
+    alphas=None,
+    fig=None,
+    ax=None,
```

Comment:
Related to above. Would it make more sense to just always pass in an `ax` object? This way the function is only adding to an existing object instead of creating a `fig` etc. I feel the creation of a `fig` here adds very little value unless this function is expanded to make multiple pie plots, export/save, or do other fig based operations.

Reply:
We should keep ax as an optional input and remove fig, this matches the API for the rest of the niceplots functions

Reply:
When matching the interface I realize now I was looking at `plotColoredLine`, which passes in a Figure for adding the colorbar. I'll remove the fig from here but keep ax optional to match other functions. Does that work for everyone?

Reply:
Thats fine with me for now.